### PR TITLE
use maxfw package

### LINKED ISSUE #9: Switch from absolute to relative path references in config.py
https://github.com/IBM/MAX-Audio-Embedding-Generator/blob/master/config.py#L26-L27 both contain a reference to `/workspace/assets/...`. While the code works within a Docker image (where the file system root is `/workspace`),  it fails if it is run outside the container:
```
tensorflow.python.framework.errors_impl.NotFoundError: /workspace/assets; No such file or directory
```

The configuration setting should be changed to use a relative path, e.g. `assets/...`.

### PR DESCRIPTION
**Pull request by bdwyer2**
- Use `maxfw`
- Cleaned up code style
- Add style checks
- Use relative paths in `config.py`, resolves #9

### CODE REVIEW

File: core/model.py [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 
-class ModelWrapper(object):
+class ModelWrapper(MAXModelWrapper):
     """
     Contains core functions to generate embeddings and classify them.
-    Also contains any helper function required. 
+    Also contains any helper function required.
     """
 
+    MODEL_META_DATA = model_meta
```
Comment (djalova) [inline core/model.py]: I think the metadata should just be defined here instead of the config.

Comment (bdwyer2) [reply core/model.py]: Do you think I should add `from config import MODEL_NAME, MODEL_LICENSE` or should I define those variable here?

Comment (djalova) [reply core/model.py]: Just define those here. I don't think those variables are used that much anyway.

File: app.py [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
-app.config.from_object('config')
-# load override config if exists
-if 'APP_CONFIG' in os.environ:
-    app.config.from_envvar('APP_CONFIG')
-api.init_app(app)
-
-if __name__ == '__main__':
-    app.run(host='0.0.0.0')
\ No newline at end of file
+max_app = MAXApp()
```
Comment (djalova) [inline app.py]: The API title, desc, and version should be passed in

File: api/predict.py [hunk 3]
```diff
@@ -1,58 +1,39 @@
-from flask_restplus import Namespace, Resource, fields
+from flask_restplus import fields
 from werkzeug.datastructures import FileStorage
 from werkzeug.exceptions import BadRequest
-from config import MODEL_META_DATA
-from core.backend import ModelWrapper
+from core.model import ModelWrapper
 import os
+from maxfw.core import MAX_API, PredictAPI
```
Comment (djalova) [inline api/predict.py:5]: Nit: The imports here could be organized better

**Comment by djalova** [comment]
LGTM

File: config.py [hunk 4]
```diff
@@ -1,27 +1,16 @@
...
 
 # Flask-restplus settings
-RESTPLUS_MASK_SWAGGER=False
+RESTPLUS_MASK_SWAGGER = False
 
 # Application settings
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
+API_TITLE = 'Model Asset Exchange Microservice'
```
Comment (djalova) [inline config.py]: API title should be updated here

**Pull request merged**